tests/Resolver/TaxType/CanadaTaxTypeResolverTest.php

```diff
@@ -156,30 +197,56 @@ public function testResolver($resolver)
         $novaScotiaAddress->expects($this->any())
             ->method('getAdministrativeArea')
             ->will($this->returnValue('CA-NS'));
+        $manitobaAddress = $this
+            ->getMockBuilder('CommerceGuys\Addressing\Model\Address')
+            ->getMock();
+        $manitobaAddress->expects($this->any())
+            ->method('getCountryCode')
+            ->will($this->returnValue('CA'));
+        $manitobaAddress->expects($this->any())
+            ->method('getAdministrativeArea')
+            ->will($this->returnValue('CA-MB'));
 
         // Nova Scotia store, Ontario customer.
         $context = $this->getContext($ontarioAddress, $novaScotiaAddress);
         $results = $resolver->resolve($taxable, $context);
-        $result = reset($results);
+        $result  = reset($results);
         $this->assertInstanceOf('CommerceGuys\Tax\Model\TaxType', $result);
         $this->assertEquals('ca_on_hst', $result->getId());
```

Comment:
Yes. The current logic is basic, and it's assumed that for all other use cases you'd implement custom resolvers.

#34 discusses ways to identify different types of products and expand our default resolvers to cover more use cases.